1-org/envs/shared/remote.tf

```diff
@@ -19,18 +19,19 @@
-  billing_account                               = data.terraform_remote_state.bootstrap.outputs.common_config.billing_account
-  default_region                                = data.terraform_remote_state.bootstrap.outputs.common_config.default_region
-  project_prefix                                = data.terraform_remote_state.bootstrap.outputs.common_config.project_prefix
-  folder_prefix                                 = data.terraform_remote_state.bootstrap.outputs.common_config.folder_prefix
-  networks_step_terraform_service_account_email = data.terraform_remote_state.bootstrap.outputs.networks_step_terraform_service_account_email
-  org_step_terraform_service_account_email      = data.terraform_remote_state.bootstrap.outputs.organization_step_terraform_service_account_email
-  bootstrap_folder_name                         = data.terraform_remote_state.bootstrap.outputs.common_config.bootstrap_folder_name
-  cloud_build_private_worker_pool_id            = try(data.terraform_remote_state.bootstrap.outputs.cloud_build_private_worker_pool_id, "")
-  required_groups                               = data.terraform_remote_state.bootstrap.outputs.required_groups
+  org_id                                            = data.terraform_remote_state.bootstrap.outputs.common_config.org_id
+  parent_folder                                     = data.terraform_remote_state.bootstrap.outputs.common_config.parent_folder
+  parent                                            = data.terraform_remote_state.bootstrap.outputs.common_config.parent_id
+  billing_account                                   = data.terraform_remote_state.bootstrap.outputs.common_config.billing_account
+  default_region                                    = data.terraform_remote_state.bootstrap.outputs.common_config.default_region
+  project_prefix                                    = data.terraform_remote_state.bootstrap.outputs.common_config.project_prefix
+  folder_prefix                                     = data.terraform_remote_state.bootstrap.outputs.common_config.folder_prefix
+  networks_step_terraform_service_account_email     = data.terraform_remote_state.bootstrap.outputs.networks_step_terraform_service_account_email
+  org_step_terraform_service_account_email          = data.terraform_remote_state.bootstrap.outputs.organization_step_terraform_service_account_email
+  bootstrap_folder_name                             = data.terraform_remote_state.bootstrap.outputs.common_config.bootstrap_folder_name
+  cloud_build_private_worker_pool_id                = try(data.terraform_remote_state.bootstrap.outputs.cloud_build_private_worker_pool_id, "")
+  required_groups                                   = data.terraform_remote_state.bootstrap.outputs.required_groups
+  organization_step_terraform_service_account_email = data.terraform_remote_state.bootstrap.outputs.organization_step_terraform_service_account_email
```

Comment:
duplicated with 
```
org_step_terraform_service_account_email          = data.terraform_remote_state.bootstrap.outputs.organization_step_terraform_service_account_email
```

Reply:
will remove the duplicate local var in next commit